lib/CompilerImpl/ClangLLVM/LLVMInstanceManager.cpp

```diff
@@ -97,9 +97,14 @@ void LLVMInstanceManager::initializeLLVM() {
   llvm::initializeTarget(passRegistry);
   // For codegen passes, only passes that do IR to IR transformation are
   // supported.
-  llvm::initializeCodeGenPreparePass(passRegistry);
+#if LLVM_VERSION_MAJOR > 18
```

Comment:
`<`

Reply:
was fixed in [2368c34](https://github.com/skeru/libVersioningCompiler/pull/5/commits/2368c342b7a0e107307786d0a23c85ad1678ccbf)